cmd/cni-ipvlan-vpc-k8s-tool/cni-ipvlan-vpc-k8s-tool.go

```diff
@@ -128,24 +128,18 @@ func actionAllocate(c *cli.Context) error {
 		index := c.Int("index")
 		ipBatchSize := c.Int64("ip_batch_size")
 		res, err := aws.DefaultClient.AllocateIPsFirstAvailableAtIndex(index, ipBatchSize)
-		if err != nil {
```

Comment:
why is this block removed? if err != nil, then the allocation failed. is the assumption that res is empty if err is != nil?

Reply:
Yeah, there's no reason to print the error and then also return it and that's all it does, if err != nil then res is empty and will skip the for loop anyways, so we can just return the error as-is at the end